sfge/src/main/java/com/salesforce/exception/SfgeRuntimeException.java

```diff
@@ -1,19 +1,30 @@
 package com.salesforce.exception;
 
+import com.salesforce.telemetry.TelemetryUtil;
+
 public abstract class SfgeRuntimeException extends RuntimeException {
     public SfgeRuntimeException() {
         super();
+        TelemetryUtil.postExceptionTelemetry(
+                this.getClass().getSimpleName() + ": No message available");
     }
 
     public SfgeRuntimeException(Throwable cause) {
         super(cause);
+        String telemetryMessage =
+                String.format("%s: %s", this.getClass().getSimpleName(), cause.getMessage());
+        TelemetryUtil.postExceptionTelemetry(telemetryMessage);
```

Comment:
Is there a way we can store the stacktrace of causedBy too?

Reply:
Maybe. I'll see what I can do.